presto-orc/src/main/java/com/facebook/presto/orc/stream/OrcInputStream.java

```diff
@@ -254,69 +253,7 @@ public String toString()
                 .add("source", source)
                 .add("compressedOffset", compressedSliceInput.position())
                 .add("uncompressedOffset", current == null ? null : current.position())
-                .add("compression", compressionKind)
+                .add("decompressor", decompressor)
```

Comment:
maybe add a `toString` to the compressor implementations which returns `snappy` or `zlib` and then change this to `decompressor.map(Object::toString).orElse("none")`